class-swiftype-client.php

```diff
@@ -191,7 +191,7 @@ public function create_or_update_document( $engine_id, $document_type_id, $docum
 	* @return array An array of true/false elements indicated success or failure of the creation or update of each individual document
 	*/
 	public function create_or_update_documents( $engine_id, $document_type_id, $documents ) {
-		$url = $this->endpoint . 'engines/' . $engine_id . '/document_types/' . $document_type_id . '/documents/bulk_create_or_update';
+		$url = $this->endpoint . 'engines/' . $engine_id . '/document_types/' . $document_type_id . '/documents/bulk_create_or_update_verbose';
```

Comment:
I haven't tested it, but I believe this change will break this code: https://github.com/swiftype/swiftype-wordpress/blob/6b9dfe5393d67d2277c1399ddce9205f77b75369/class-swiftype-plugin.php#L478. Because the verbose endpoint returns `true` on success and a string of some sort on failure, and in PHP a non-empty string is truthy. So `if( $record )` will always be `true`.